programming-languages/README.md

```diff
@@ -24,4 +24,29 @@
 ### Tools
 
 * [Compiler Explorer](https://godbolt.org/): Online C++ compiler. It lets you compile small code snippets and look at the generated assembly code. It supports different compilers across various architectures ranging from x86 to Arduinos.
-* [JetBrains CLion](https://www.jetbrains.com/clion/): One of the best C++ IDEs. CLion works on CMake making it easy to use with roscpp. 
\ No newline at end of file
+* [JetBrains CLion](https://www.jetbrains.com/clion/): One of the best C++ IDEs. CLion works on CMake making it easy to use with roscpp. 
+
+
+## Python
+A high level, interpreted, dead-simple, reads-like-english programming language used for a multitude of tasks, from Machine Learning/Data Analysis to deploying Full-Stack Websites.
+
+The [official website](https://www.python.org/) offers further resources, including talks, podcasts, conferences and the like.
+
+### Tutorials
+
+* [Sentdex Python Tutorial Series](https://www.youtube.com/playlist?list=PLQVvvaa0QuDe8XSftW-RAxdo6OmaeL85M): A collection of short videos that deal with everything from installing python, through loops, control flow, functions, upto usage of the package manager pip. Beyond this, the tutorial deals with specific use cases which may be skipped. It is highly recommended that you code along with the videos.
+
+
+### IDEs
+
+* [JetBrains' PyCharm](https://www.jetbrains.com/pycharm/): Fully featured Python IDE, offering powerful autocomplete, virtual environment support, comprehensive debugging, and notebook support. Users of other JetBrains IDEs will feel right at home.
```

Comment:
Apply for Jetbrains student pack if you have college email-id. It gives you free access to premium products.

Reply:
I think you can add a down point also. Like its very slow and may run with observable lags on low speced pc @aayush-fadia